site/_docs/reference.md

```diff
@@ -2739,6 +2739,9 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b | FORMAT_TIMESTAMP(string timestamp)             | Formats *timestamp* according to the specified format *string*
 | s | GETBIT(value, position)                        | Equivalent to `BIT_GET(value, position)`
 | b o | GREATEST(expr [, expr ]*)                    | Returns the greatest of the expressions
+| h s | HEX(binary)                                  | Converts *binary* into a hexadecimal string. For example, hex(x'6162') returns '6162'
+| h s | HEX(bigint)                                  | Converts *bigint* into a shortened hexadecimal string without leading zeros. For example, hex(10) returns 'A'
```

Comment:
the explanation for shortened makes sense, but perhaps the documentation should say "with leading zeros removed"

Reply:
Thanks, I will refine it.